yktrace/Cargo.toml

```diff
@@ -11,7 +11,7 @@ elf = "0.0"
 fallible-iterator = "0.2"
 ykpack = { path = "../ykpack" }
 lazy_static = "1.3"
-hwtracer = { git = "https://github.com/softdevteam/hwtracer" }
+hwtracer = { path = "../../hwtracer" }
```

Comment:
This shouldn't be here, right?

Reply:
Oops. That's not needed I don't think. Will check.